src/bsp_types/basic_bsp_structures.rs

```diff
@@ -80,6 +80,12 @@ pub struct BuildTargetIdentifier {
     pub uri: Uri,
 }
 
+impl From<&str> for BuildTargetIdentifier {
+    fn from(uri: &str) -> Self {
```

Comment:
w nazwie jest uri, a string może przecież nie być prawidłowym uri

Reply:
Ta funkcja jest używana tylko w testach Patryka, więc usunęłam to z tego PR i poprawimy to w PR z testami